webui/tests/requirements.txt

```diff
@@ -0,0 +1 @@
+pexpect
```

Comment:
suggestion: we are also using the docker package in the WebUI tests module can we add that as well?